crds/tests/test_certify.py

```diff
@@ -757,6 +757,36 @@ def certify_AsdfCertify_valid():
     >>> doctest.ELLIPSIS_MARKER = '...'
     """
 
+def certify_roman_valid_asdf():
+    """
```

Comment:
It is possible to add a traditional docstring description at the top of these new tests before the code starts (see `test_certify_check_rmap_updates` for an example).  Maybe we should add some prose to each of these explaining what they test, and also mention that they're testing official Roman requirements?  Just so people know that they can't be refactored away even if they appear to be similar to other existing tests.  @nden do you think we should document the specific requirement number here in the test or is that unnecessary detail?

Reply:
you dont need to necessarily document the requirement number, but definitely add some simple prose explaining what is being tested, and what the input and expected output should be :)

Reply:
I will add comments today.

Reply:
Comments added.